### ISSUE TITLE
TracerWarning: Converting a tensor to a Python number might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!

### ISSUE THREAD

**Issue by fcqfcq** [pytorch/pytorch#36454]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

**Comment by fcqfcq** [comment]
/input/.py:210: TracerWarning: Converting a tensor to a Python integer might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  padding=self.padding, stride=self.stride, groups=N).contiguous().view(N,oC,H,W)
/input/.py:168: TracerWarning: Converting a tensor to a Python integer might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  out = F.conv_transpose2d(x, weight, padding=self.padding, stride=self.stride, groups=N)
/input/.py:131: TracerWarning: torch.tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.
  C= torch.tensor( feature.shape).index_select(0,torch.tensor([1])).item()
/input/.py:131: TracerWarning: Converting a tensor to a Python integer might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  C= torch.tensor( feature.shape).index_select(0,torch.tensor([1])).item()
/input/.py:131: TracerWarning: Converting a tensor to a Python number might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  C= torch.tensor( feature.shape).index_select(0,torch.tensor([1])).item()

**Comment by YIQIHYM** [comment]
如何解决啊，大神能给个方法吗。

**Comment by YIQIHYM** [comment]
升级onnx版本，就解决
